src/test/java/uk/gov/register/presentation/functional/FunctionalTest.java

```diff
@@ -41,6 +45,26 @@ public void shouldConsumeMessageFromKafkaAndShowAsLatest() throws Exception {
 
     }
 
+    @Test
+    public void appSupportsCORS() {
```

Comment:
@om-sharma pointed out to me in slack that this won't work as a resource test as the functionality is provided by a Filter instead.  The full DropwizardAppRule test is the only way to test this functionality.  Sadly that means you need to connect to a real database and kafka server.

I'd rather have very few functional tests and focus them on ensuring that the whole app is glued together correctly.  The CrossOriginFilter is part of jetty so I think we can be quite confident that it will work.